scripts/regression.sh

```diff
@@ -3,12 +3,42 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPT_DIR}/lib/robot.sh"
 
-# FW_FILE and DEVICE_IP are required for full regression
-check_env_variable "FW_FILE"
-check_env_variable "DEVICE_IP"
+if [ $CONFIG == "qemu" ]; then
+  export RTE_IP=127.0.0.1
+  export SNIPEIT_NO="no"
+  compatibility_tests=(
+  "dasharo-compatibility/custom-boot-menu-key.robot"
```

Comment:
Ok, I am not a huge fan of such solution.
Ideally, we simply execute the ./rgression.sh as we do for any other platform, and only the required tests are executed, thanks to the correctly set test flags.

Reply:
Isn't this what happens in this script? If we set `CONFIG` to `qemu`, only some of the tests are executed (the ones that were listed in `regression-qemu.sh`, so I assume those can run on QEMU)

Reply:
@PLangowski  But this should be resolved by properly setting support flags in platform-configs/qemu.robot, not be the if-else here.